### PR TITLE
Print unexpected response in the logs

### PR DESCRIPTION
**Pull request by gaganawhad**
@benhutton I am curious to learn more about what sort of response we are getting from facebook. I am hoping this will help.

### CODE REVIEW

**Comment by gaganawhad** [comment]
[Update Facebook Share Count Service to Handle API Failures Better](https://trello.com/c/bLY1f6hR/2925-update-facebook-share-count-service-to-handle-api-failures-better)

File: app/wrappers/wakes/facebook_metrics_wrapper.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def share_counts
           hash[r['id']] = r.fetch('share').fetch('share_count')
         rescue KeyError
           # Some urls can have no 'share' keys. This is generally in the case when the url isn't valid.
-          Rails.logger.warn "Share count not received for #{r['id']}. Possibly an invalid url?"
+          Rails.logger.warn "Share count not received for #{r['id']}. Response is: #{r}"
```
Comment (benhutton) [inline app/wrappers/wakes/facebook_metrics_wrapper.rb]: @gaganawhad what kind of object is `r`? There are response objects out there that could be thousands and thousands of characters. I don't think getting all that in our logs is helpful or useful. Do you know what the average response object is?

Comment (gaganawhad) [inline app/wrappers/wakes/facebook_metrics_wrapper.rb]: @benhutton That's a good question. Here is the documentation: https://developers.facebook.com/docs/graph-api/reference/v2.5/url. What do you think?

Comment (gaganawhad) [inline app/wrappers/wakes/facebook_metrics_wrapper.rb]: Oh, and I don't mean to keep that log that way for good. May be we can improve what gets logged once we learn what is happening? (`id` being `nil` seems strange). 

Not directly related, but we are running an older version of the api. We should probably create a task to have it be upgraded at an appropriate time.

**Comment by gaganawhad** [comment]
Thanks @benhutton !

**Pull request merged**